### DISCUSSION TITLE
How to change default logo??

### DISCUSSION THREAD

**Discussion by moshiuzzaman** [payloadcms/payload#277] [Q&A] [answered]
How to change default logo in navbar without add any custom navbar???

**Comment by richardvanbergen** [comment, marked as answer]
See `graphics.Icon` and `graphics.Logo` here: https://payloadcms.com/docs/admin/component. They have their own key so you don't have to swap out the full nav 👍

**Reply by shaneheyworth** [reply]
Hi there, this link is broken; I tried following what you guys did in your example as per the docs (but the example wasn't for logos) and it just loads a blank page. Can this be re-hashed?

**Reply by DanRibbens** [reply]
Here is the correct link: https://payloadcms.com/docs/admin/components

Also we have a blog post about this too. 
https://payloadcms.com/blog/white-label-admin-ui

Let us know if you have trouble or need anything else.
